frameworks/elastic/docs/upgrade.md

```diff
@@ -21,3 +21,111 @@ $ dcos package install elastic --options=/path/to/options.json
 The command above will trigger the install of the new Elastic version. You can follow the upgrade progress by making a REST request identical to the one used to follow the progress of a configuration upgrade. See the Configuring section for more information.
 
 Note: The upgrade process will cause all of your Elastic node processes to restart.
+
+## Rolling Upgrades
+
+By default, the update strategy is `serial`, which waits for each node to be ready before moving on to the next one. But the elasticsearch node being ready doesn't mean all the shards in the indexes are ready. This may be an appropriate strategy for test clusters or clusters where writes can be suspended during the upgrade/update process, but for production clusters you may want to use the `safe` strategy.
+
+In the `safe` strategy, the operator manually triggers the upgrade for each node, pausing to make sure the deployment is going as planned using `dcos elastic plan show`. The following rolling upgrade process is based on Elastic's own (recommendations for rolling upgrades)[https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html]. 
+
+After starting the upgrade process, the plan will await your input to continue:
+```
+$ dcos elastic plan show
+deploy (WAITING)
+├─ master (WAITING)
+│  ├─ master-0:[node] (WAITING)
+│  ├─ master-1:[node] (WAITING)
+│  └─ master-2:[node] (WAITING)
+├─ data (WAITING)
+│  ├─ data-0:[node] (WAITING)
+│  └─ data-1:[node] (WAITING)
+├─ ingest (WAITING)
+│  └─ ingest-0:[node] (WAITING)
+└─ coordinator (WAITING)
+   └─ coordinator-0:[node] (WAITING)
+```
+
+Steps 1 and 2 are the same as in Elastic's guide:
+1. Disable shard allocation
```

Comment:
- periods at the end of the list items.
- can number each `1.` and markdown will generate the correctly numbered list.